src/components/AccountInfo.js

```diff
@@ -18,17 +18,24 @@ export class AccountInfo extends React.Component {
                     userBalancesIsLoading
                 }
                 header={header}
+                style={{ padding: "0px 10px", border: "1px solid #ffad00", borderRadius: "0px 0px 5px 5px"}}
```

Comment:
ideally these all should live in the style.js

Reply:
in first round i fixed white on white balace nums & copy icon

Reply:
second round:
added the orange bg to the account info inside the topnav
coming up: third round w separators